src/components/Dropdown.vue

```diff
@@ -52,50 +68,63 @@ export default {
 
 <style lang="scss" scoped>
 .dropdown {
-  display: inline-block;
   position: relative;
 
-  .display {
-    text-align: center;
-    position: relative;
-    color: $standard_font_color;
-    font-size: 0.75rem;
-    background-color: $light_color;
-    padding: 0.6rem 2.2rem 0.6rem 0.85rem;
-    line-height: 0.9rem;
-    min-height: 2.2rem;
+  > button {
     display: flex;
     align-items: center;
+    padding: 0.2rem 0.4rem;
+    font-size: 0.75rem;
+    color: $standard_font_color;
+    border-radius: 2.5rem;
 
-    .rounded {
-      border-radius: 0.25rem;
+    &.active {
+      background-color: $article_content_color;
     }
+  }
+
+  .dropdown-item {
+    font-size: 0.75rem;
+    min-width: 6.5rem;
+    padding: 0.6rem 0.8rem;
+    background-color: $actions_ribbon_background_color;
+    color: $standard_font_color;
+    box-shadow: inset 0 0 0.1rem $article_content_color;
+    border-radius: unset;
+    outline: none;
```

Comment:
You can remove this by using ButtonPlain

Reply:
@stanislav-slavov clean up the styles, `outline: none` is still there

Reply:
Removed it, thanks.